src/App.js

```diff
@@ -3,9 +3,11 @@ import {Route, Switch} from 'react-router';
 import Categories from 'components/Categories/Categories';
 import Nav from '../src/components/Nav';
 import Favourite from 'components/Favourite/Favourite';
+import ShoppingCard from 'components/ShoppingCard/ShoppingCard';
 
 function App() {
     const [favourites, setFavourites] = useState([]);
+    const [baskets, setBaskets] = useState([]);
```

Comment:
baskets sounds like ther were several baskets, but actually its serveral items in one basket. maybe `basketItems` would be more concise here

Reply:
i changed baskets => basketItems, favourites => favouriteIems